modules/xmpp/strophe.jingle.js

```diff
@@ -204,17 +202,17 @@ class JingleConnectionPlugin extends ConnectionPlugin {
         }
         case 'transport-replace':
             logger.info('(TIME) Start transport replace', now);
-            Statistics.analytics.sendEvent(
-                TRANSPORT_REPLACE_START,
-                { value: now });
+            Statistics.analytics.sendEvent(createJingleEvent(
+                ACTION_JINGLE_TR_RECEIVED,
+                { value: now }));
 
             sess.replaceTransport($(iq).find('>jingle'), () => {
                 const successTime = window.performance.now();
 
                 logger.info('(TIME) Transport replace success!', successTime);
-                Statistics.analytics.sendEvent(
-                    TRANSPORT_REPLACE_SUCCESS,
-                    { value: successTime });
+                Statistics.analytics.sendEvent(createJingleEvent(
```

Comment:
It's very likely not really the scope of this PR but it's definitely exacerbating the following problem for me:

Given that `Statistics` is a "class" and `Statistics.analytics` is static, I'm not sure why we're sprinkling our source code with long lines of `Statistics.analytics.sendEvent` plus its arguments which are not always trivial. I'd suggest having a mere function with a name of less length like `sendAnalytics`.

Reply:
I don't think we want "analytics" to be static. We have a class already, we might as well instantiate it. But I would leave this for another PR.

As for shortening the name, I would like this, but I'm not sure what the best way would be, because analytics now exports a set of functions: sendEvent, sendUIEvent, sendTrackEvent, sendPageEvent, sendOperationalEvent. Do we expose them all with shorter names, or do we just add sendAnalytics as a shortcut for the (most commonly used) sendEvent?

Reply:
I will think about this some more, because I'm not convinced anymore that we need to expose these multiple functions from analytics.

Reply:
I'm removing the sendXXXEvent methods and only leaving sendEvent. And also adding a Statistics.sendAnalytics() shortcut.